components/match2/commons/get_match_group_copy_paste_wiki_base.lua

```diff
@@ -37,7 +37,7 @@ function wikiCopyPaste._getMaps(bestof)
 	local map = '{{Map|map=}}'
 	local out = ''
 	for i = 1, bestof do
-		out = out .. '\n    |map' .. i .. '=' .. map
+		out = out .. '\n	|map' .. i .. '=' .. map
```

Comment:
I think this one may be intended

Reply:
i excluded these modules from the replace, seems like 1 occurence slipped through
double checking everything